### DISCUSSION TITLE
Interest in sdc-tools Users' Workshop 2023

### DISCUSSION THREAD

**Discussion by violetacln** [sdcTools/UserSupport#249] [Polls]
We invite you to decide together whether to organize a new SDC-tools Users' _Workshop_ in Wiesbaden, in September 2023, accompanying the UNECE conference on SDC (26-28 September, https://indico.un.org/event/1002843/ ).
We remind you that this is in addition to our regular User Group _meetings_ which always take place at the beginning of the year. 
Please, feel free to comment and propose interesting topics or formats for the workshop, too.

**Comment by ReiffertMichel** [comment]
Hello Everyone, 
I would be very happy to have a Workshop here in Wiesbaden, since most of the participants are already travelling for the conference. 
But easy for me to say, since I dont have to travel and stay in a hotel for those days ;-)

Regards,
Michel
